examples/kitchensink/app-reversed/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Reversed example in frint",
   "main": "index.js",
   "scripts": {
-    "build": "../../../node_modules/.bin/webpack"
```

Comment:
this also means, we get rid of relative paths in npm scripts, fixing Windows issue at the same time.